docs/C/ssids.rst

```diff
@@ -136,6 +136,52 @@ Basic Subroutines
    Provided for backwards comptability, users are encourage to use 64-bit ptr
    in new code.
 
+.. c:function:: void spral_ssids_analyse_topology(bool check, int n, int *order, const int64_t *ptr, const int *row, const double *val, void **akeep, const struct spral_ssids_options *options, struct spral_ssids_inform *inform, int nregions, const struct spral_numa_region *regions)
```

Comment:
Suggest to do this like the documentation for the _ptr32 version; refer to the original `spral_ssids_analyse` and only elaborate on the additional parameters, to avoid duplication that makes it harder to maintain and read.